tests/Http/SignupTest.php

```diff
@@ -6,10 +6,38 @@
 use Rogue\Models\Post;
 use Rogue\Models\User;
 use Rogue\Models\Signup;
+use Rogue\Services\Registrar;
 use DoSomething\Gateway\Blink;
 
 class SignupTest extends TestCase
 {
+    /**
+     * Helper function to mock NS API calls and receive a user.
+     */
+    public function mockNorthstarApiCallAndReturnUser($northstarId = null)
+    {
+        // Mock the Northstar API calls.
+        $this->mock(Registrar::class)
+            ->shouldReceive('find')
+            ->andReturnUsing(function($northstarId) {
+                $mockNorthstarUser = new \StdClass;
```

Comment:
Let's make this a [`NorthstarUser`](https://github.com/DoSomething/gateway/blob/15d3201b904d34d4220193938c0d4dce067f32dc/src/Resources/NorthstarUser.php) instead of a `StdClass`, in case we type-check this anywhere:

```php
->andReturnUsing(function($northstarId) {
    return new NorthstarUser([
        'id' => $northstarId ? $northstarId : $this->faker->northstar_id,
        'first_name' => $this->faker->firstName,
        // ...
    ]);
}
```